aggregator/README.md

```diff
@@ -0,0 +1,54 @@
+# notifications-aggregator project
```

Comment:
Do we want to keep that generated readme?

Reply:
No :)